packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -2833,18 +2833,6 @@ class TestFlutterDevice extends FlutterDevice {
   Stream<Uri> _observatoryUris;
 }
 
-class ThrowingForwardingFileSystem extends ForwardingFileSystem {
```

Comment:
Isn't this used (line 2508)?

Reply:
oh woops, I was filtering my analysis errors :(

Reply:
Fixed